axelor-base/src/main/java/com/axelor/apps/base/exceptions/IExceptionMessage.java

```diff
@@ -417,4 +417,9 @@
   /* Scheduler */
   public static final String QUARTZ_SCHEDULER_ENABLED = /*$$(*/
       "The scheduler service is enabled." /*)*/;
+
+  /* Geonames import */
+  public static final String NO_DATA_FILE_FOUND = /*$$(*/ "No file found on %s." /*)*/;
+  public static final String URL_NOT_SPECIFIED = /*$$(*/
+      "Please put GeoNames URL in Base config" /*)*/;
```

Comment:
```suggestion
  public static final String GEONAMES_URL_NOT_SPECIFIED = /*$$(*/
      "Please fill in GeoNames URL in Base application configuration." /*)*/;
```

Please don't forget to replace the constant name in https://github.com/axelor/axelor-open-suite/pull/6539/files#diff-722a09ab5419ce355b1c625826f2d64be7293d290c9b9da40d17070aa2537dc4R452 too and refresh i18n.

🇫🇷  translation: "Veuillez remplir l'URL GeoNames dans la configuration de l'application Base."